variables.tf

```diff
@@ -363,3 +363,36 @@ variable "custom_endpoint_certificate_arn" {
   description = "ACM certificate ARN for custom endpoint."
   default     = ""
 }
+
+variable "auto_tune_enabled" {
+  type        = bool
+  description = "Whether to enable autotune."
+  default     = false
+}
+
+variable "auto_tune_rollback_settings" {
+  type        = string
+  description = "Whether to roll back to default Auto-Tune settings when disabling Auto-Tune."
+  default     = "NO_ROLLBACK"
+
+  validation {
+    condition     = contains(["DEFAULT_ROLLBACK", "NO_ROLLBACK"], var.auto_tune_rollback_settings)
+    error_message = "Valid values: DEFAULT_ROLLBACK or NO_ROLLBACK."
+  }
+}
+
+variable "auto_tune_starting_time" {
+  type        = string
+  description = "Date and time at which to start the Auto-Tune maintenance schedule in RFC3339 format. Time should be in the future."
+  default     = null
+}
+
+variable "auto_tune_cron_schedule" {
```

Comment:
@dm3ch 
pls add here the default value; otherwise, this is a required variable.

Reply:
These values are not used by default (if `auto_tune_enabled = false`), so requiring them is not a best option.
And there's no default value in provider as far as I remember.

So I see 2 possible options how it can be done:
- Add some default value, which we would think optimal
- Try to add a validation rule that would check if `auto_tune_enabled = true` and require variable in this case

What do you think would be better?

Reply:
As far as I remember validation condition can't refer to other vars, so the second option is impossible. :(

So to be honest I don't know how to better workaround this. Do you have any thoughts?